trin-core/src/jsonrpc/service.rs

```diff
@@ -30,6 +30,7 @@ pub fn launch_jsonrpc_server(
     trin_config: TrinConfig,
     infura_project_id: String,
     portal_tx: UnboundedSender<PortalJsonRpcRequest>,
+    live_server_tx: tokio::sync::mpsc::Sender<bool>,
```

Comment:
I also considered making the `live_server_tx` an `Option`. In the main boot process, I've never run into this issue, and can't imagine a scenario where you need the jsonrpc server immediately (as we do in the tests) and don't end up waiting around however many ms it takes for the server to boot.